### PR TITLE
Change Endpoint docs link to security-docs repo

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by bmorelli25** [comment]
Something funky is going on with the preview. I'm going to rebuild to see if that fixes things:

@elasticmachine, run elasticsearch-docs/doc rebuild

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
The doc build still isn't picking up the changes. Need to force a rebuild.

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-docs/doc rebuild

**Comment by bmorelli25** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by bmorelli25** [comment]
There it is! Docs were wrong, gotta update. Now we wait.

File: conf.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ repos:
     kibana-cn:            https://github.com/elasticsearch-cn/kibana.git
     logstash:             https://github.com/elastic/logstash.git
     logstash-docs:        https://github.com/elastic/logstash-docs.git
+    security-docs:        https://github.com/elastic/security-docs.git
```
Comment (bmorelli25) [inline conf.yaml]: This repo doesn't exist?
https://github.com/elastic/security-docs

Comment (DonNateR) [reply conf.yaml]: @bmorelli25: Might be due to the repo currently being private. Let me switch to the public and see if that fixes the preview.

**Comment by DonNateR** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by DonNateR** [comment]
@bmorelli25: Looks like we're good.  http://docs_1766.docs-preview.app.elstc.co/guide/en/endpoint/guide/current/index.html

**Pull request merged**